scripting/weaponmodels.sp

```diff
@@ -37,27 +50,31 @@
 
 #pragma semicolon 1
 
+
 #include <sourcemod>
 #include <sdktools>
 #include <sdkhooks>
 
 #define PLUGIN_NAME "Custom Weapon Models"
-#define PLUGIN_VERSION "1.2"
+#define PLUGIN_VERSION "1.5"
+#define ENGINE_NMRIH 19
```

Comment:
This should not be necessary, use Engine_SDK2013 instead

Reply:
noted.